my-app/src/components/beachspeech/BSGameLobby.js

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+import "../../stylesheets/beachspeech.css"
+import Island from "./Island"
+import CenterColumn from "./CenterColumn"
+import RightColumn from "./RightColumn"
+import Developers from "./Developers"
+import playerData from "../../data/beachspeech/playerData.json"
+
+/* Beach Speech Game Lobby */
+class BSGameLobby extends React.Component {
+    constructor() {
+        super()
```

Comment:
no semicolon and on line 26

Reply:
For React, they don't need any semi-colons, and I am always confused about where to put the semi colons because of the ludicrous amounts of brackets. The tutorial was doing it this way, so idk what the given standard is.

Reply:
yah java script in general doesnt require it but it will help just to identify the end of functions a little better so i like to still use it. But we didn't really talk about it withe everyone so its up to u if u want to still use it